docs/resources/sks_kubeconfig.md

```diff
@@ -0,0 +1,70 @@
+---
+page_title: "Exoscale: exoscale_sks_kubeconfig"
+description: |-
+  Provides an Exoscale SKS cluster kubeconfig.
+---
+
+# exoscale\_sks\_kubeconfig
+
+Provides an Exoscale [SKS][sks-doc] Kubeconfig resource. This can be used to create a configuration file (Kubeconfig) to interact with SKS clusters.
+
+
+## Example Usage
+
+```hcl
+locals {
+  zone = "de-fra-1"
+}
+
+resource "exoscale_sks_cluster" "prod" {
+  zone    = local.zone
+  name    = "prod"
+  version = "1.20.2"
+
+  labels = {
+    env = "prod"
+  }
+}
+
+resource "exoscale_sks_kubeconfig" "prod_admin" {
+  zone = local.zone
+
+  ttl_seconds = 3600
+  early_renewal_seconds = 300
+  cluster_id = exoscale_sks_cluster.prod.id
+  user = "kubernetes-admin"
+  groups = ["system:masters"]
+}
+
+output "kubeconfig" {
+  value = exoscale_sks_kubeconfig.prod_admin.kubeconfig
+  sensitive = true
+}
+```
+
+
+## Arguments Reference
+
+* `zone` - (Required) The name of the [zone][zone] of the target SKS cluster.
+* `cluster_id` - (Required) The ID of the target SKS cluster.
+* `ttl_seconds` - The number of seconds after initial issuing that the Kubeconfig will become invalid.
+* `early_renewal_seconds` - If set, the resource will consider the Kubeconfig to have expired the given number of seconds before its actual ca certificate or client certificate expiry time. This can be useful to deploy an updated Kubeconfig in advance of the expiration of its internal current certificate. Note however that the old certificate remains valid until its true expiration time since this resource does not (and cannot) support certificate revocation. Note also that this advance update can only be performed should the Terraform configuration be applied during the early renewal period.
```

Comment:
```suggestion
* `early_renewal_seconds` - If set, the resource will consider the Kubeconfig to have expired the given number of seconds before its actual ca certificate or client certificate expiry time. This can be useful to deploy an updated Kubeconfig in advance of the expiration of its internal current certificate. Note however that the old certificate remains valid until its true expiration time since this resource does not (and cannot) support certificate revocation. Note also that this advance update can only be performed should the Terraform configuration be applied during the early renewal period (default: 0).
```